### PR TITLE
Move HeaderNavigation and OrdinoFooter components to ordino

### PR DESCRIPTION
**Pull request by oltionchampari**
Closes Caleydo/tdp_bi_bioinfodb#1200

### Summary 
see Caleydo/ordino#274

### CODE REVIEW

**Comment by oltionchampari** [comment]
Build fails because it requires Caleydo/ordino#274

File: src/pages/DatasetsPage.tsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
-import {HeaderNavigation} from './components/HeaderNavigation';
-import {OrdinoFooter} from './components/OrdinoFooter';
+import {OrdinoFooter} from 'ordino';
 import {Row, Col, Card, Container} from 'react-bootstrap';
-
+import {HeaderNavigation} from 'ordino';
```
Comment (thinkh) [inline src/pages/DatasetsPage.tsx:4]: Imports can be combined.

File: src/pages/Error404Page.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import {HeaderNavigation} from './components/HeaderNavigation';
-import {OrdinoFooter} from './components/OrdinoFooter';
+import {HeaderNavigation} from 'ordino';
+import {OrdinoFooter} from 'ordino';
```
Comment (thinkh) [inline src/pages/Error404Page.tsx:3]: Imports can be combined.

File: src/pages/FeaturesPage.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import {HeaderNavigation} from './components/HeaderNavigation';
-import {OrdinoFooter} from './components/OrdinoFooter';
+import {HeaderNavigation} from 'ordino';
+import {OrdinoFooter} from 'ordino';
```
Comment (thinkh) [inline src/pages/FeaturesPage.tsx:3]: Imports can be combined.

File: src/pages/HelpPage.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import {HeaderNavigation} from './components/HeaderNavigation';
-import {OrdinoFooter} from './components/OrdinoFooter';
+import {HeaderNavigation} from 'ordino';
+import {OrdinoFooter} from 'ordino';
```
Comment (thinkh) [inline src/pages/HelpPage.tsx:3]: Imports can be combined.

File: src/pages/NewsPage.tsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import {HeaderNavigation} from './components/HeaderNavigation';
-import {OrdinoFooter} from './components/OrdinoFooter';
+import {HeaderNavigation} from 'ordino';
+import {OrdinoFooter} from 'ordino';
```
Comment (thinkh) [inline src/pages/NewsPage.tsx:3]: Imports can be combined.

File: src/pages/PublicationPage.tsx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import {HeaderNavigation} from './components/HeaderNavigation';
-import {OrdinoFooter} from './components/OrdinoFooter';
+import {HeaderNavigation} from 'ordino';
+import {OrdinoFooter} from 'ordino';
```
Comment (thinkh) [inline src/pages/PublicationPage.tsx:3]: Imports can be combined.

**Review by thinkh — CHANGES_REQUESTED** [review]
* Just some minor optimizations
* We can probably also remove the ordino.svg file here as it was moved to the _ordino_ repo. Please check for possible usages here again.

**Review by thinkh — APPROVED** [review]
Looks good!

**Pull request merged**